vmm/src/lib.rs

```diff
@@ -88,7 +89,8 @@ use vmm_config::net::{NetworkInterfaceConfig, NetworkInterfaceConfigs, NetworkIn
 use vmm_config::vsock::{VsockDeviceConfig, VsockDeviceConfigs, VsockError};
 use vstate::{Vcpu, Vm};
 
-const DEFAULT_KERNEL_CMDLINE: &str = "reboot=k panic=1 pci=off nomodules 8250.nr_uarts=0";
+const DEFAULT_KERNEL_CMDLINE: &str = "reboot=k panic=1 pci=off nomodules 8250.nr_uarts=0 \
+                                      i8042.noaux i8042.nomux i8042.nopnp i8042.dumbkbd";
```

Comment:
The default kernel command line is specified in the README.md or some other documentation pages. Also, for this feature to work I guess that the users of Firecracker will need to update their own kernel command line because I don't think they are using our default. So to make sure people know about this I would do 2 things:
1. Add an entry in the changelog so the change to the default command line is **visible**.
2. Also specify in the action `SendCtrlAltDel` documentation that the command line is altered. I am expecting people to look at the documentation when they want to use this action and to be fairly easy to realize that the `boot_args` will also need to be updated.

Reply:
Done. Updated changelog and added thorough documentation for the new API action.